charter.html

```diff
@@ -236,28 +259,29 @@ <h2 id=coordination>Coordination</h2>
 <a href=https://www.ecma-international.org/>Ecma</a>,
 <a href=https://ietf.org/><abbr
 title="Internet Engineering Task Force">IETF</abbr></a>, and elsewhere,
-and will transition spec proposals to them when they’re ready for the
-standards track. Groups most likely to be close partners are listed
-below, but this group is expected to coordinate with other groups as
-relevant.
+and will transition <a href=#work-items>Work Items</a> to them when
+they’re ready for the standards track. Groups most likely to be close
+partners are listed below, but this group is expected to coordinate with
+other groups as relevant.
 
 <h3 id=w3c-coordination>W3C Groups</h3>
 
 <dl>
 <dt><a href=https://www.w3.org/Privacy/IG/>Privacy Interest Group (PING)</a>
 <dd>This group will coordinate with PING and will take into
-consideration outputs of PING when evaluating proposals.
+consideration outputs of PING when
+evaluating <a href=#proposals>Proposals</a> and Work Items.
 <dt><a href=https://www.w3.org/2011/webappsec/>Web Application Security
 Working Group (WebAppSec)</a>
 <dd>WebAppSec is expected to be the main destination for transitioning
-this group’s independent Deliverables to the standards track.
+this group’s independent Work Items to the standards track.
 <dt><a href=https://www.w3.org/community/wicg/>Web Platform Incubator
 Community Group (WICG)</a>
-<dd>WICG is expected to be a major source of Deliverables for this
+<dd>WICG is expected to be a major source of Work Items for this
 group.
-<p class=note>Only privacy-related WICG work items which have the
-support of at least two <a href=#implementer>implementers</a> are
-eligible for this group to take up as Deliverables.
+<p class=note>Only privacy-related WICG proposals which have the support
```

Comment:
I wonder if we should remove this note given we have the two implementer support called out in other places.

Reply:
It's a note because the normative text that sets this requirement is elsewhere. I think it's mildly helpful as a reminder, but I think it'd be okay to remove too.

Reply:
@erik-anderson?

Reply:
I have no preference.